### PR TITLE
docs: fixed indentation in examples

### PR DESCRIPTION
**Pull request by MarcusNoble**
Couple examples had the indentation wrong.

### CODE REVIEW

File: docs/kubernetes/ingress-usage.md [hunk 1]
```diff
@@ -115,10 +115,10 @@ spec:
 rules:
```
Comment (AlexanderYastrebov) [inline docs/kubernetes/ingress-usage.md:115]: @MarcusNoble Thank you for the fix.

`rules:` seems to be indented incorrectly as well.

Comment (MarcusNoble) [reply docs/kubernetes/ingress-usage.md]: Good spot. Updated :)

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**